benchmarking/nursery/benchmark_neuralband/baselines.py

```diff
@@ -192,43 +188,3 @@ def conv_numeric_only(margs) -> dict:
         random_seed=method_arguments.random_seed,
     ),
 }
-
```

Comment:
Removed code

Reply:
why did you remove it. we no longer want this?